libs/SmartStore/src/com/salesforce/androidsdk/smartstore/app/SmartStoreSDKManager.java

```diff
@@ -412,28 +420,32 @@ public void setupUserStoreFromDefaultConfig() {
         }
     }
 
+    @NonNull
     @Override
-    protected LinkedHashMap<String, DevActionHandler> getDevActions(final Activity frontActivity) {
-        LinkedHashMap<String, DevActionHandler> devActions = super.getDevActions(frontActivity);
+    protected Map<String, DevActionHandler> getDevActions(
```

Comment:
Here's another Collections implementation to interface downstream of Kotlin conversion.  The rest of this file is inspector cleanup.